include/libtorrent/alert_types.hpp

```diff
@@ -117,10 +117,10 @@ namespace libtorrent
 		virtual std::string message() const override;
 
 		// The peer's IP address and port.
-		tcp::endpoint ip;
+		tcp::endpoint const ip;
```

Comment:
I don't think so
```cpp
void pop_alerts(std::vector<alert*>* alerts);
alert* wait_for_alert(time_duration max_wait);
```
btw, putting this constant has a significant impact in the swig interface, it does not generate the "setter"